apps/meteor/client/views/room/ShareLocation/MapLibreMap.tsx

```
@@ -0,0 +1,168 @@
+		return () => {
+			resizeObsRef.current?.disconnect();
+			window.removeEventListener('resize', onWinResize);
+			map.remove();
+			mapRef.current = null;
+			markerRef.current = null;
+			trailRef.current = null;
+			trailCoordsRef.current = [];
+		};
+	    map.on('error', (e) => {
+			console.error('MapLibre error:', e);
+		});
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)